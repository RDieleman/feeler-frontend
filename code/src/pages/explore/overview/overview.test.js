import React from "react";
import {shallow} from "enzyme";
import ExploreOverviewPage from "./overview.page";

it("Expect to render ExploreOverview page", () => {

    expect(shallow(
        <ExploreOverviewPage

        />)).toMatchSnapshot();
});