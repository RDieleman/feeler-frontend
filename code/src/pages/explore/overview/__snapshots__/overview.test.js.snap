// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render ExploreOverview page 1`] = `ShallowWrapper {}`;
